pkg/apis/componentconfig/v1alpha1/types.go

```diff
@@ -93,13 +93,13 @@ type KubeProxyConntrackConfiguration struct {
 type KubeProxyConfiguration struct {
 	metav1.TypeMeta `json:",inline"`
 
-	// featureGates is a comma-separated list of key=value pairs that control
-	// which alpha/beta features are enabled.
-	//
-	// TODO this really should be a map but that requires refactoring all
+	// TODO FeatureGates really should be a map but that requires refactoring all
```

Comment:
without an empty line this ends up in docs

Reply:
Are you saying we should add a comment so people know to preserve whitespace?

Reply:
This is what it currently generates:

```yaml
# enableProfiling enables profiling via web interface on /debug/pprof handler. Profiling handlers will be handled by metrics server.
enableProfiling: false
# featureGates is a comma-separated list of key=value pairs that control which alpha/beta features are enabled.
featureGates: ""
# healthzBindAddress is the IP address and port for the health check server to serve on, defaulting to 0.0.0.0:10256
healthzBindAddress: 0.0.0.0:10256
```

Reply:
ok, this looks good. There was another PR in the last days which added such a newline. are TODO lines filtered out?

Reply:
there's "commentLines" and "secondClosestCommentLines" in gengo, hence the need for the newline :-)